src/components/film-genre/film-genre.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+
+import './film-genre.css';
+
+class FilmGenre extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 2,
+    };
+    this.onClickToggleActive = this.onClickToggleActive.bind(this);
+  }
+
+  onClickToggleActive(doActiveItem) {
+    const newActiveItem = doActiveItem;
+    this.setState(() => {
+      const newState = { activeItem: newActiveItem };
+      return { ...newState };
+    });
+  }
+
+  render() {
+    const {
+      activeItem,
+    } = this.state;
+    return (
+      <div className="film-genre">
+        <button
+          type="button"
+          className={`film-genre-item  ${activeItem === 1 ? 'active' : null} `}
+          onClick={() => this.onClickToggleActive(1)}
+        >
+          ALL
+        </button>
+        <button
+          type="button"
+          className={`film-genre-item  ${activeItem === 2 ? 'active' : null} `}
```

Comment:
in case `activeItem` is not equal `2` className returns - `film-genre-item  null `